app/src/main/java/app/revanced/integrations/returnyoutubedislike/ReturnYouTubeDislike.java

```diff
@@ -254,9 +254,11 @@ private static boolean updateDislike(AtomicReference<Object> textRef, boolean is
             }
             replacementSpannable = newSpannableWithDislikes(oldSpannable, voteData);
         } else {
-            String leftSegmentedSeparatorString = ReVancedUtils.isRightToLeftTextLayout() ? "\u200F|  " : "|  ";
+            final boolean useCompactLayout = SettingsEnum.RYD_USE_COMPACT_LAYOUT.getBoolean();
+            // if compact layout, use a "half space" character
+            String middleSegmentedSeparatorString = useCompactLayout ? "\u2009 • \u2009" : "  •  ";
```

Comment:
Does this really make a difference?

![image](https://user-images.githubusercontent.com/13122796/211216638-b132f644-75e1-401b-8a0d-e36f9c6768fe.png)

![image](https://user-images.githubusercontent.com/13122796/211216644-3b62d282-98ea-443a-8fd0-744455080141.png)

Reply:
46 pixels, vs 50 pixels.  Not much of a difference, no.  But it is more 'compact'.